### ISSUE TITLE
CppParameter doesn't expose default value

### ISSUE THREAD

**Issue by EgorBo** [xoofx/CppAst#3]
A small feature request:
```c
#define DEFAULTMAXDEPTH 1024
bool allocateCapacity(size_t len, size_t maxdepth = DEFAULTMAXDEPTH);
```
Currently it's not possible to extract that default value for `maxdepth` = `1024`

**Comment by xoofx** [comment]
Oh right, actually working on it

**Comment by xoofx** [comment]
Should be fixed via ba670ef
Available shortly in `0.2.0`
